StripeConnect/StripeConnectTests/Internal/Webview/ConnectComponentWebViewControllerTests.swift

```diff
@@ -0,0 +1,259 @@
+//
+//  ConnectComponentWebViewControllerTests.swift
```

Comment:
This was renamed from `ConnectComponentWebViewTests` but I added enough new tests to it that GH no longer recognized the move (highlighted in comments below)

Or you can diff these locally with this:
```sh
git fetch mludowise/refactor_to_vc
git difftool origin/master:StripeConnect/StripeConnectTests/Internal/Webview/ConnectComponentWebViewTests.swift origin/mludowise/refactor_to_vc:StripeConnect/StripeConnectTests/Internal/Webview/ConnectComponentWebViewControllerTests.swift
```